File: javap/control_statement/book.java
package javap.control_statement;

class Book {
    public Boolean data = false;

    void method() {
        if (data) {
            System.out.println("Control Condtion executed on true");
        } else {
            System.out.println("Control Condtion executed on false");
        }
    }

    public static void main(String args[]) {
        Book b1 = new Book();
        b1.method();
        b1.data = true;
        b1.method();
    }
}

File: pythonp/control_statement/book.py
class Book:
    data = True

    def method(self):
        if self.data == False:
            print("Control Condtion executed on true")
        else:
            print("Control Condtion executed on false")


b1 = Book()
b1.method()
b1.data = False
b1.method()
